tiledb/sm/c_api/tiledb_enum.h

```diff
@@ -158,6 +158,10 @@
     TILEDB_FILTER_TYPE_ENUM(FILTER_BYTESHUFFLE) = 9,
     /** Positive-delta encoding filter. */
     TILEDB_FILTER_TYPE_ENUM(FILTER_POSITIVE_DELTA) = 10,
+    /** MD5 checksum filter. */
+    TILEDB_FILTER_TYPE_ENUM(FILTER_CHECKSUM_MD5) = 12,
```

Comment:
11 instead of 12?

Reply:
I will add a comment to this line for future clarity but 12 is correct, 11 is used internally for encryption filter: https://github.com/TileDB-Inc/TileDB/blob/1e6fa1d1316cbfcb2e6ae055a04f852a25e4a773/tiledb/sm/enums/filter_type.h#L54